.github/workflows/test.yml

```diff
@@ -35,26 +35,6 @@ jobs:
 
     # Build caches for release-canary workflow
 
-    - if: github.ref == 'refs/heads/master'
-      name: upload elements artifact
-      uses: actions/upload-artifact@v2
-      with:
-        name: elements
-        path:
-          packages/elements/dist/
-
-    - if: github.ref == 'refs/heads/master'
-      name: upload elements-react artifact
-      uses: actions/upload-artifact@v2
-      with:
-        name: elements-react
-        path:
-            packages/elements-react/dist/
-
-    - if: github.ref == 'refs/heads/master'
-      name: upload elements-angular artifact
-      uses: actions/upload-artifact@v2
-      with:
-        name: elements-angular
-        path:
-            packages/elements-angular/elements/dist/
+    #- if: github.ref == 'refs/heads/master'
```

Comment:
Do we want to upload build artifacts for non master branches? In which cases we need this?

Reply:
Whops, forgot to uncomment this one